prExplorer/client-app/src/app/common/form/CustomTextArea.tsx

```diff
@@ -0,0 +1,23 @@
+import { useField } from 'formik';
+import React from 'react';
+import { Form, Label } from 'semantic-ui-react';
+
+interface CustomTextAreaProps {
+    placeholder: string;
+    name: string;
+    rows: number;
+    label?: string;
+}
+
+export default function CustomTextInput(customTextAreaProps: CustomTextAreaProps) {
+    const [field, meta] = useField(customTextAreaProps.name);
+    return (
+        <Form.Field error={meta.touched && !!meta.error}>
+            <label>{customTextAreaProps.label}</label>
+            <textarea {...field} {...customTextAreaProps} />
+            {meta.touched && meta.error ? (
```

Comment:
It seems that this part for displaying validation errors could be extracted as a separate component. It depends on the type of `meta`, but I guess you can make it work